### PR TITLE
Kinga_straszkiewicz_TD

### PR DESCRIPTION
**Pull request by kingastraszkiewicz**

### CODE REVIEW

File: Straszkiewicz_TechWr.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+Then, you have to write a key word. What follows is the list of sentences in which that keyword appears and the number of those sentences.
+You also have an information about the number of positive and negative words in those sentences and the statistical analysis of the sentiment in those sentences.
+
+![alt text](https://github.com/kingastraszkiewicz/Project-Sentiment-Analysis/blob/master/2019-06-10_21h23_16.png)
+
+Additionally, the list is accompanied by a pie chart presenting statistical information.
+
+![alt text](https://github.com/kingastraszkiewicz/Project-Sentiment-Analysis/blob/master/key.png)
+
+##Altering Files
```
Comment (MarcinFortuna-UG) [inline Straszkiewicz_TechWr.md]: Space missing after ##

File: Straszkiewicz_TechWr.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+To run, download a zip file:
+
+https://github.com/kingastraszkiewicz/Project-Sentiment-Analysis/blob/master/Projekt.zip
+
+
+## User Guide
+After launching the tool, there appears the Menu displaying the contents of the programme and the numbers, which are ascribed to the options.
+The first thing that you have to do is to pick one of the two texts meant for analysis.
+
+![alt text](https://github.com/kingastraszkiewicz/Project-Sentiment-Analysis/blob/master/2019-06-10_17h49_01.png)
```
Comment (MarcinFortuna-UG) [inline Straszkiewicz_TechWr.md]: Your images do not render properly ;( You are linking the so-called "blob", please try a normal link instead.

**Review by MarcinFortuna-UG — CHANGES_REQUESTED** [review]

**Review by MarcinFortuna-UG — APPROVED** [review]
Dear Kinga,

unfortunately I still can't see the images ;( However, I will already merge it, since you've had several attempts and I'm sure you would like to concentrate on something else now ;) 

In the meantime, you may want to try to fix the problem with images for your own sake later on.

**Comment by kingastraszkiewicz** [comment]
I'm sorry, are they visible now? It was probably because my repository was Private.

**Comment by MarcinFortuna-UG** [comment]
Yes they are! It looks great! :)

**Comment by kingastraszkiewicz** [comment]
Thank you very much :))

**Pull request merged**